sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -501,6 +501,13 @@ aditof::Status CameraItof::setMode(const uint8_t &mode) {
         }
     }
 
+    // If a Dynamic Mode Switching sequences has been loaded from config file then configure ADSD3500
+    if (m_configDmsSequence.size() > 0) {
+        this->adsd3500setEnableDynamicModeSwitching(true);
+        this->adsds3500setDynamicModeSwitchingSequence(m_configDmsSequence);
```

Comment:
Checking both return values might be helpful. In case of first fails, the second is not required, and the second one can provide different outcomes.

Reply:
Yes, thanks! I have handled the errors.